packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -42,118 +37,78 @@ interface AnimalInventoryProps {
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
 
+  const { inventory, isLoading } = useAnimalInventory();
+
   const animalsColumns = useMemo(
     () => [
       {
-        id: 'name',
+        id: 'identification',
         label: t('ANIMAL.ANIMAL_IDENTIFICATION').toLocaleUpperCase(),
-        format: (d: AnimalOrBatchData) => (
+        format: (d: AnimalInventory) => (
           <Cell
             kind={CellKind.ICON_TEXT}
-            text={d.name || d.identifier || null}
-            icon={AnimalIcon}
-            subtext={
-              isMobile
-                ? `${d.default_type_id || d.custom_type_id} / ${
-                    d.default_breed_id || d.custom_breed_id
-                  }`
-                : null
-            }
+            text={d.identification}
+            icon={d.icon}
+            subtext={isMobile ? `${d.type} / ${d.breed}` : null}
+            highlightedText={d.count > 1 ? d.count : null}
           />
         ),
       },
       {
-        id: isMobile ? null : 'default_type_id',
+        id: isMobile ? null : 'type',
         label: t('ANIMAL.ANIMAL_TYPE').toLocaleUpperCase(),
-        format: (d: AnimalOrBatchData) => (
-          <Cell kind={CellKind.PLAIN} text={d.default_type_id || d.custom_type_id} />
-        ),
+        format: (d: AnimalInventory) => <Cell kind={CellKind.PLAIN} text={d.type} />,
       },
       {
-        id: isMobile ? null : 'default_breed_id',
+        id: isMobile ? null : 'breed',
         label: t('ANIMAL.ANIMAL_BREED').toLocaleUpperCase(),
-        format: (d: AnimalOrBatchData) => (
-          <Cell kind={CellKind.PLAIN} text={d.default_breed_id || d.custom_breed_id} />
-        ),
+        format: (d: AnimalInventory) => <Cell kind={CellKind.PLAIN} text={d.breed} />,
       },
       {
         id: isMobile ? null : 'groups',
         label: t('ANIMAL.ANIMAL_GROUPS').toLocaleUpperCase(),
-        format: (d: AnimalOrBatchData) => (
-          <Cell
-            kind={CellKind.HOVER_PILL_OVERFLOW}
-            items={d.groups && d.groups.map((group) => group.name)}
-          />
+        format: (d: AnimalInventory) => (
+          <Cell kind={CellKind.HOVER_PILL_OVERFLOW} items={d.groups} />
         ),
         sortable: false,
       },
       {
-        id: 'farm_id',
-        label: t('ANIMAL.ANIMAL_LOCATIONS').toLocaleUpperCase(),
-        format: (d: AnimalOrBatchData) => <Cell kind={CellKind.PLAIN} text={d.farm_id} />,
-      },
-      {
-        id: 'Visit Record',
+        id: 'path',
         label: '',
-        format: (d: AnimalOrBatchData) => <Cell kind={CellKind.RIGHT_CHEVRON_LINK} path="/" />,
+        format: (d: AnimalInventory) => <Cell kind={CellKind.RIGHT_CHEVRON_LINK} path={d.path} />,
         columnProps: {
-          style: { width: '40px', padding: `0 ${isMobile ? 8 : 12}px` },
+          style: { padding: `0 ${isMobile ? 8 : 12}px` },
         },
         sortable: false,
+        width: '40px',
```

Comment:
Just wondering, does this not work as a `columnProp`?

Reply:
Will remove all of this. I was having problems with width being overridden -- I don't know why I thought this would do something different 😂 .

Making the fixed layout based on mobile is what my problem was I believe.